### PR TITLE
Add notes for configuring OpenBSD syslog receivers

### PR DESCRIPTION
**Pull request by dspruell**

### CODE REVIEW

**Comment by jdillard** [comment]
LGTM. Here is the edited page: https://deploy-preview-23--pfsense-docs.netlify.com/monitoring/copying-logs-to-a-remote-host-with-syslog.html

I'll wait for you approval @jim-p.

File: source/monitoring/copying-logs-to-a-remote-host-with-syslog.rst [hunk 1]
```diff
@@ -16,12 +16,12 @@ Setup Syslog on the Logging Host
...
 
 First, configure the syslog server to accept remote connections which
-means running it with the ``-a <subnet>`` flag.
+means running it with the ``-a <subnet>`` or similar flag.
 
 FreeBSD
 ^^^^^^^
 
-On FreeBSD, edit /etc/rc.conf and add this line::
+On FreeBSD, edit **/etc/rc.conf** and add this line::
```
Comment (jim-p) [inline source/monitoring/copying-logs-to-a-remote-host-with-syslog.rst]: Should be backticks (` ``foo`` `), not bold

File: source/monitoring/copying-logs-to-a-remote-host-with-syslog.rst [hunk 2]
```diff
@@ -43,9 +43,9 @@ Now, edit **/etc/syslog.conf** and add a block at the bottom::
   +pfSense
   *.*                /var/log/pfsense.log
 
-Where pfSense is the hostname of the pfSense firewall. An entry may also
+Where *pfSense* is the hostname of the pfSense firewall. An entry may also
 need to be added in **/etc/hosts** for that system, depending on the DNS
-setup. Logs may be split separate files. Use the /etc/syslog.conf file
+setup. Logs may be split separate files. Use the **/etc/syslog.conf** file
```
Comment (jim-p) [inline source/monitoring/copying-logs-to-a-remote-host-with-syslog.rst]: Should be backticks (` ``foo`` `), not bold

File: source/monitoring/copying-logs-to-a-remote-host-with-syslog.rst [hunk 3]
```diff
@@ -59,12 +59,27 @@ Now restart syslog::
...
+^^^^^^^
+
+The configuration for OpenBSD is similar to FreeBSD, with the following notes:
+
+#. The option to accept remote syslog events is ``-u``.
+#. This option may be enabled using *rcctl(8)*::
+
+    rcctl set syslogd flags -u
+
+#. To restart the syslogd service, use *rcctl(8)* too::
```
Comment (jim-p) [inline source/monitoring/copying-logs-to-a-remote-host-with-syslog.rst]: No need for "too" here. Actually everything after the comma could be removed.

**Pull request merged**